apps/rule-manager/client/src/ts/components/RuleContent.tsx

```diff
@@ -146,6 +150,18 @@ export const RuleContent = ({
 				<EuiSpacer size="s" />
 				<EuiFormRow
 					label={<Label text="Pattern" required={true} />}
+					labelAppend={
+						<Link
+							to={
+								isTestingRule
+									? `/rule/${ruleData?.draft.id}`
+									: `/rule/${ruleData?.draft.id}/test`
+							}
+						>
+							<EuiIcon type="bug" />
```

Comment:
Would the bug icon look better on the right side, to be consistent with the 'Preview' and 'Discard unpublished changes' icons above? Up to you though!

Reply:
Good suggestion – resolving as the UI has changed since this was asked, the relevant bit no longer exists!